bridge-patterns/bridge-frontend
# project setp

- What does this PR do?
This PR updates the codebase for the crisis text line. Adds a number of pages that can be tweaked to address our specific needs.
- What Trello task is related to this PR?
Trello card is [here](https://trello.com/c/2zkPKUtX)
- How do I test out this PR?
Git clone this repo cd bridge-frontend/crisis-text-line Then run npm install to get all the necessary dependencies Afterwards run npm start to test out the server.
- What is the expected output?
![image](https://user-images.githubusercontent.com/1590213/84990552-f50ba280-b14d-11ea-86a4-25ccaa724a56.png)
![image](https://user-images.githubusercontent.com/1590213/84990570-fccb4700-b14d-11ea-8a3a-d1907b49d586.png)
